File: jv/insertar.java
public class insertar {
    
    public static void main(String[] args) {
        
        int[] num = {1,2,3,4,5}, newnum = new int[5];
        int newElement = 76, posicion = 2, j= 0;

        for (int idx = 0; idx < num.length; idx++) {
            if ( posicion == idx + 1 ) {
                newnum[idx] = newElement;
                j++;
            }
            else{
                newnum[idx] = num[idx-j];
            }
            
        }

        for(int n:newnum){
            System.out.print(n + ", ");
        }

    }
}

File: c#/insertar/insertar.cs
using System;

class Insertar
{
    public static void Main(string[] args)
    {
        Random rand = new Random();
        int numeroAdd = rand.Next(1, 50);
        int[] numeros = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
        int[] newnum = new int[10];
        int posicion = 5, j = 0;

        for (int i = 0; i < 10; i++)
        {
            if (i == posicion)
            {
                newnum[i] = numeroAdd;
                j++;
            }
            else
            {
                newnum[i] = numeros[i - j];
            }
        }

        for (int i = 0; i < 10; i++)
        {
            Console.WriteLine(newnum[i]);
        }

    }
}
